### ISSUE TITLE
 Pickled model instance's Django version is not specified.

### ISSUE THREAD

**Issue by Koed00** [Suor/django-cacheops#153]
This warning has been popping up since Django 1.8

```python
local/lib/python2.7/site-packages/cacheops/query.py:257: RuntimeWarning: Pickled model instance's Django version is not specified.
  results = pickle.loads(cache_data)
```
Obviously it's associated with this 

'Pickling models and querysets across different versions of Django isn’t officially supported (it may work, but there’s no guarantee). An extra variable that specifies the current Django version is now added to the pickled state of models and querysets, and Django raises a RuntimeWarning when these objects are unpickled in a different version than the one in which they were pickled.'.

I've checked several times, but I've not been unpickling between Django versions

**Comment by Suor** [comment]
Doesn't look like this is cacheops specific. Maybe you upgraded django an
have old cache? You can also try to pickle/unpicke some model in django
shell.

**Comment by Koed00** [comment]
It might be related to django-pickling. I'll poke around a little.

**Comment by Suor** [comment]
You haven't mentioned django-pickling and this is cacheops repo. It could be related though, you can try disabling it.

**Comment by Koed00** [comment]
I totally forgot it was in the project. I disabled django-pickling, invalidated everything and cleared the cache, but it is still producing the same warning. 
It only happens once, when the server starts and the first user logs in.  With or without a filled cache.
I'm not too worried about it and it's probably very specific. It might even be a Django bug.
I'll report back if I find anything new.

**Comment by Suor** [comment]
Anyway, there is no way I can do soething about it unless you make a test from it.

**Comment by Koed00** [comment]
Agreed. I'm just sidetracked on another project right now.

**Comment by Koed00** [comment]
Turns out it wasn't enough to just remove django-pickling from installed_apps , cacheops was importing it anyway. After uninstalling it and clearing Redis, the warnings disappear.
Django now puts a version key in with the models, that is not pickled by django-pickling.

**Comment by Suor** [comment]
Python imports upickle function specified in pickle, it's not related to cacheops. I still can't see how this warnings could be generated since version check should not be triggered when unpickling django-pickling pickles.

**Comment by Koed00** [comment]
Maybe when django-pickling unpickled models get pickled and unpickled again by another process?

**Comment by Suor** [comment]
Process doesn't matter. Why should it?

**Comment by Suor** [comment]
Looking a bit more into it I can't see how cacheops can cause this. It doesn't modify pickling process, so the error should be caused by something else. `django-pickling` also can't cause this, because when used they completely bypass normal unpickling process.

The possibility is that you use outdated Django in some additional task: background, cron, queue job or older instance.

**Comment by vangale** [comment]
I saw this warning on an app that used django-pickling but didn't use django-cacheops. I removed django-pickling from INSTALLED_APPS and the warning went away.

The warning was being generated when running unittests so there were no background tasks, crons, queues, etc. Also, the warning wasn't for mismatched Django versions. The warning was that the pickled object didn't contain the expected version data.

Btw, here's the Django commit that added this pickle version checking: https://github.com/django/django/commit/42736ac
